### DISCUSSION TITLE
OC 0.8.7 + 2013 / 2017 MacBook Pro SMBIOS

### DISCUSSION THREAD

**Discussion by MultimediaLucario** [MultimediaLucario/Lenovo-ThinkPad-T440s#24] [Announcements]
![repository-open-graph-template new](https://user-images.githubusercontent.com/72415505/193061418-2920edc3-48f6-4397-b9da-e88fe475b626.png)

#
## Updated to OpenCore Version 0.8.7! 💻 🖥

This release comes with **four** EFI folders, one that supports up to **macOS 11 Big Sur** and older which contains the 2013 13" MacBook Pro SMBIOS, one that supports **macOS 12 Monterey** which contains the **2017** 13" MacBook Pro SMBIOS, one that supports **macOS 13 Ventura**, and an EFI folder that is optimized for ThinkPads with an **Intel Core I5-4210U**! **Please note that the SMBIOS for Ventura is still in **BETA** so do not use this for your everyday productivity!** 

<hr /><em>This discussion was created from the release <a href='https://github.com/MultimediaLucario/Lenovo-ThinkPad-T440s/releases/tag/OC-0.8.7'>OC 0.8.7 + 2013 / 2017 MacBook Pro SMBIOS</a>.</em>

**Comment by marco9566** [comment]
Hello, I'm French and I use your efi for my T440s laptop, you did a wonderful job.  However, I encounter a problem.  I send you the screenshot.
![373ED5A8-3BE9-4AA1-A7B0-3932B2DA9E17](https://user-images.githubusercontent.com/120490096/208321978-02b21a8b-220a-4430-b5da-6104166e84f6.jpeg)

**Comment by MultimediaLucario** [comment]
Ok, which EFI folder did you use and what is the CPU that is equipped with your T440s?

**Comment by marco9566** [comment]
hi ! i am equipped with intel i5 4300u, the installation works with (efi monterey) however i have a question i wanted to try sidecar with my wireless ipad and it does not work, is it normal? do you plan to develop ventura in a stable version? thank you very much for your job very well done congratulations you are a great chef

**Reply by MultimediaLucario** [reply]
Sidecar does not work normally. And the error could easily be fixed by resetting the NVRAM.

**Comment by marco9566** [comment]
Thank you so much ! 
- I can't find how to reset nvram?
- do you intend to continue working on the Ventura version?

  your work is fabulous!  Thank you so much !

**Reply by MultimediaLucario** [reply]
It's a little bit tricky to work on macOS Ventura Support given the fact that my T440s is shot. I will repurchase one eventually. For right now Ventura development is on hold.

**Reply by MultimediaLucario** [reply]
As for resetting the NVRAM, open up the OpenCore Configurator App and then mount the EFI to open the Config.plist. Go to Misc, then select Tools and then select the enabled toggle for CleanNVRAM.efi. Once that's done save the file and restart you computer. Then when you see the boot picker select CleanNVRAM and then restart your ThinkPad T440s once more. Now, the error would disappear.
